www/json/connectionConfig.json

```diff
@@ -0,0 +1,15 @@
+{
+    "connectUrl": "http://192.168.1.238:8080",
```

Comment:
Remove from PR since it is related to your IP address.

Reply:
Done.